examples/conservative-raster/triangle_and_lines.wgsl

```diff
@@ -0,0 +1,17 @@
+[[stage(vertex)]]
+fn vs_main([[builtin(vertex_index)]] vertex_index: u32) -> [[builtin(position)]] vec4<f32> {
```

Comment:
ok, so we aren't really using vertex attributes here, good! Let's remove the star from the table.